examples/plot_visualize_bad_epochs.py

```diff
@@ -14,32 +14,23 @@
 # sphinx_gallery_thumbnail_number = 2
 
 # %%
-# First, we download the data from OpenfMRI. We will download the tarfile,
-# extract the necessary files and delete the tar from the disk
+# First, we download the data from OpenfMRI which is hosted on OpenNeuro.
+# We will do this using ``openneuro-py``.
```

Comment:
Could you add a line here how to install it using `pip` ?

Reply:
also, it should be added as an optional dependency to https://github.com/autoreject/autoreject/blob/master/README.rst